modules/fileout_netcdf/FONcTransform.cc

```diff
@@ -509,7 +345,8 @@ void FONcTransform::transform_dap2(ostream &strm)
 
     stax = nc_set_fill(_ncid, NC_NOFILL, &current_fill_prop_vaule);
```

Comment:
I didn't review the code when this line was added. That's why it might be a good idea to ask somebody to review the code. It seems that this purpose is for performance. However, with recent empty chunk value case for GHRSST, if the whole chunk is all covered with fillvalue, set a real fillvalue may have the performance gain. Again, this is the same issue as passing the compression ratio/shuffle from upstream file, which I did an experiment at 
https://github.com/oPENDAP/libdap4/tree/esdis-580
and https://github.com/OPENDAP/bes/tree/esdis-580